plugins/embed/gui-themes/themes/zeit-solo/item/base.dust

```diff
@@ -3,26 +3,16 @@
 {! Attention! Opening tag in headerContainer … !}
 {<headerContainer}
 	<div class="liveblog-post">
-		<div class="post-header">
```

Comment:
this are changes on the themes, should also be propagated to the html generation? or when in production